buildpipeline/pipeline.json

```diff
@@ -37,7 +37,7 @@
             "PB_DockerTag": "centos-6-c8c9b08-20174310104313",
             "PB_BuildArguments": "-buildArch=x64 -Release -stripSymbols -RuntimeOS=rhel.6 -- /p:PortableBuild=false",
             "PB_BuildTestsArguments": "-buildArch=x64 -Release -SkipTests -Outerloop -RuntimeOS=rhel.6 -- /p:ArchiveTests=true /p:EnableDumpling=true",
```

Comment:
I'm not entirely sure it is needed but just to be sure can we also pass `PortableBuild=false` to the PB_BuildTestsArguments as well.

Reply:
Thanks @weshaggard . Sure I've added “/p:PortableBuild=false” to PB_BuildTestsArguments as well